graphql/resolvers.go

```diff
@@ -104,14 +121,21 @@ func (r *hostResolver) Volumes(ctx context.Context, obj *restModel.APIHost) ([]*
 		apiVolume := &restModel.APIVolume{}
 		err = apiVolume.BuildFromService(volume)
 		if err != nil {
-			return nil, errors.Wrapf(err, "error building volume '%s' from service", volId)
+			return nil, InternalServerError.Send(ctx, errors.Wrapf(err, "error building volume '%s' from service", volId).Error())
 		}
 		volumes = append(volumes, apiVolume)
 	}
 
 	return volumes, nil
 }
 
+func (r *volumeResolver) UIDisplayName(ctx context.Context, obj *restModel.APIVolume) (string, error) {
+	if obj.DisplayName != nil && *obj.DisplayName != "" {
+		return *obj.DisplayName, nil
+	}
+	return *obj.ID, nil
+}
```

Comment:
Why add an extra field and do this logic in the resolver as opposed to just doing 

`volume.DisplayName ?? volume.Id`

in the component. Since in theory we would still need both of those fields.

Reply:
unfortunately that expression won't work as expected
`"" ?? "thing"` evaluates to "" instead of "thing"

Reply:
@khelif96 I removed uiDisplayName from the schema since it seems simpler to express this from spruce